frontend/src/views/scrimmaging.js

```diff
@@ -281,14 +284,14 @@ class Scrimmaging extends Component {
         <div className="content">
           <div className="container-fluid">
             <div className="row">
-              {/* <ScrimmageRequests
+              <ScrimmageRequests
                 ref={(requests) => {
                   this.requests = requests;
                 }}
                 refresh={this.refresh}
                 episode={this.props.episode}
                 episode_info={this.props.episode_info}
-              /> */}
+              />
```

Comment:
wildly nitpicky: undo this change, cuz it's coming in the other PR, and it makes more sense there?